assets/scripts/highlight-lines.ts

```diff
@@ -21,7 +21,7 @@ function highlightLines(state, lines: [number, number][]) {
   let decorations: Range<Decoration>[] = [];
 
   // Don't show any decorations if there are no highlighted line numbers
-  if (!lines && !lines.length) return Decoration.none;
+  if (!lines || !lines.length) return Decoration.none;
```

Comment:
`lines` can be `undefined` if no highlight option is provided, so `&&` was executing the second expression.